.github/workflows/R-CMD-check-build.yaml

```diff
@@ -0,0 +1,74 @@
+on:
+  workflow_call:
+    inputs:
+      RCLR_VERSION:
+        required: true
+        type: 'string'
+    secrets:
+      GITHUB_TOKEN:
+        required: true
+
+name: R-CMD-check-build
+
+jobs:
+  R-CMD-check-build:
+    runs-on: windows-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+      R_KEEP_PKG_SOURCE: yes
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install PK-Sim
+        uses: Open-Systems-Pharmacology/Workflows/.github/actions/install-PKSIM
+
+      - name: Setup Pandoc
+        uses: r-lib/actions/setup-pandoc@v2
+
+      - name: Setup R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - name: Install rClr
+        uses: Open-Systems-Pharmacology/Workflows/.github/actions/install-rClr
+        with:
+          RCLR_VERSION: ${{ inputs.RCLR_VERSION }}
+
+      - name: Install dependencies
+        uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: |
+            rClr=local::./rClr_${{ env.RCLR_VERSION }}.zip
```

Comment:
wasn't rClr already installed with the "Install rClr" step above?

Reply:
If it is not installed here, the package resolution (done by `{pak}`) will not find rClr even though it was already installed.

I will remove the installation step from install-rClr.yml and rename it get-rClr.yml instead.